pkg/inspector/image-inspector.go

```diff
@@ -337,7 +338,7 @@ func (i *defaultImageInspector) pullImage(client *docker.Client) error {
 			}
 			go decodeDockerResponse(parsedErrors, reader)
 
-			if err := client.PullImage(imagePullOption, auth); err != nil {
+			if err = client.PullImage(imagePullOption, auth); err != nil {
```

Comment:
yes; `authErr` was always `nil`. Technically not a refactor; i can remove it from the PR if desired